var/ramble/repos/builtin/applications/tpu-device-count/application.py

```diff
@@ -0,0 +1,45 @@
+from ramble.appkit import *
+
+
+class TpuDeviceCount(ExecutableApplication):
+    """This is an example application that runs on TPUs"""
+
+    name = "tpu-device-count"
+
+    tags("tpu")
+
+    software_spec(
+        "libtpu",
+        pkg_spec="-f https://storage.googleapis.com/jax-releases/libtpu_releases.html",
+        package_manager="pip",
+    )
+
+    software_spec(
+        "jax",
+        pkg_spec="jax[tpu]",
+        package_manager="pip",
```

Comment:
This should be: `when="package_manager_family=pip"`

Reply:
addressed, PTAL